packages/backend/server/routes/publish.post.ts

```diff
@@ -7,19 +7,22 @@ import { randomUUID } from "uncrypto";
 import { setItemStream, useTemplatesBucket } from "~/utils/bucket";
 import { useOctokitInstallation } from "~/utils/octokit";
 import { generateTemplateHtml } from "~/utils/template";
+import fsSync from "fs";
+import path from "path";
 
 export default eventHandler(async (event) => {
   const origin = getRequestURL(event).origin;
-
   const {
     "sb-run-id": runIdHeader,
     "sb-key": key,
     "sb-shasums": shasumsHeader,
     "sb-comment": commentHeader,
     "sb-compact": compactHeader,
+    "sb-package-manager": packageManagerHeader,
   } = getHeaders(event);
   const compact = compactHeader === "true";
   const comment: Comment = (commentHeader ?? "update") as Comment;
+  const packageManager = packageManagerHeader || detectPackageManager(".");
```

Comment:
This is the server, how would we detect the user's packageManager from the server?